src/protos/io/defang/v1/fabric.proto

```diff
@@ -321,10 +321,18 @@ message Version {
   string pulumi_min = 4; // minimum Pulumi provider version
 }
 
+enum LogType {
+  ALL = 0;
+  RUN = 1;
+  BUILD = 2;
```

Comment:
suggest making these names more specific (e.g. LOGTYPE_ALL). protobuf enum names are global scope, these names are a little too generic and may cause conflicts/confusion in the future.

Reply:
Oh wow, I didn't know that. Thanks for the tip!

Reply:
Are these flags, ie. can they be or'ed? Should/can they be?

Reply:
Good call. I had thought about it, but felt like it was too much complexity, but I gave it a shot after your suggestion. 

[`51652c9` (#769)](https://github.com/DefangLabs/defang/pull/769/commits/51652c90ff3ce63e2357f675cf573715d3c9b099)

I think it ended up providing a much nicer interface though.